src/components/Footer.jsx

```diff
@@ -1,20 +1,175 @@
 import React from 'react'
+import { Link } from 'gatsby'
+import slackLogo from '../images/slack.svg'
+import githubLogo from '../images/github.svg'
+import meetup from '../images/meetup.svg'
 import styled from 'styled-components'
-import logo from '../images/cfc_logo_short.svg'
+import longArrowAltRight from '../images/join-us/long-arrow-alt-right.svg'
 
 const StyledFooter = styled.footer`
-  width: 14rem;
+  width: 100%;
+  padding: 1rem 0;
+  display: block;
+  margin: 0 auto;
+
+  @media (min-width: 1200px) {
+    padding: 0rem 15.25rem;
+  }
+
+  .footer-content {
+    margin: 0 auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    gap: 1rem;
+
+    @media (min-width: 1200px) {
+      width: 653.16px;
```

Comment:
let's make sure to always use `rem` with any size or spacing rules. `px` should only really ever be used with border width and border radius. And I use px for media queries as I've found inconsistent behavior with rem. 

Otherwise width, gap, margins, paddings, and pretty much every other rule should be done with rem.